c2corg_ui/static/js/editing/outingediting.js

```diff
@@ -122,7 +124,10 @@ app.OutingEditingController.prototype.successRead = function(response) {
  */
 app.OutingEditingController.prototype.prepareData = function(data) {
   // adapt the Object for what's expected on the API side + format the outing
+  this.submit = true;
```

Comment:
> I admit it would make sense to move this.submit = true; directly to the preview service caller in the parent class

Actually I have removed this ``this.submit`` flag because it was outing-specific and could be replaced by an additional argument in the outing's ``formatOuting_`` method. See 08b7274